pom.xml

```diff
@@ -191,7 +191,26 @@
                 <plugin>
                     <groupId>org.codehaus.mojo</groupId>
                     <artifactId>build-helper-maven-plugin</artifactId>
-                    <version>{version.buildhelper.plugin}</version>
+                    <version>${version.buildhelper.plugin}</version>
+                </plugin>
+                <plugin>
```

Comment:
This should be in the parent right? Not needed here

Reply:
@kenfinnigan - the version is managed in the open-api parent and used in the TCK. This plugin is not present in the smallrye-parent POM at all.

Reply:
Should've been specific, I meant the jacoco plugin defined below

Reply:
Ugh, after checking, it's not actually defined in SmallRye Parent yet.

Would you be able to submit a PR for that? https://github.com/smallrye/smallrye-parent/issues/19

Reply:
Yes, I can do that.

Reply:
Opened smallrye/smallrye-parent#21 and removed the plugin configuration from the parent of this project.

Reply:
Just released the parent.

In about an hour it should be in Central so you can update PR to use `8`

Reply:
Updated to use smallrye-parent version 8.